### PR TITLE
refactor: server.lua to server/main.lua, move to modules

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

- Moved server.lua to server/main.lua to match the client side
- Moved to modules
- Removed client/utils.lua as imports/utils.lua already does what it did
- Replace functions in favor of imports/utils.lua
- Further use of ox_lib
- Some style fixes

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -440,10 +421,10 @@ RegisterNetEvent('qb-vehicleshop:server:checkFinance', function()
 end)
 
 -- Transfer vehicle to player in passenger seat
-QBCore.Commands.Add('transfervehicle', Lang:t('general.command_transfervehicle'), {{name = 'ID', help = Lang:t('general.command_transfervehicle_help')}, {name = 'amount', help = Lang:t('general.command_transfervehicle_amount')}}, false, function(source, args)
+lib.addCommand('transfervehicle', {help = Lang:t('general.command_transfervehicle'), params = {{name = 'id', type = 'playerId', help = Lang:t('general.command_transfervehicle_help')}, {name = 'amount', type = 'number', help = Lang:t('general.command_transfervehicle_amount')}}}, function(source, args)
     local src = source
-    local buyerId = tonumber(args[1]) --[[@as number]]
-    local sellAmount = tonumber(args[2]) --[[@as number]]
+    local buyerId = args.id
```
Comment (mockdot) [inline server/main.lua:426]: I think this actually needs to use the tonumber(), have you tested it to see if it works?

Comment (mafewtm) [reply server/main.lua:426]: ox_lib already does this when parsing through the command
image

Comment (mafewtm) [reply server/main.lua:426]: quick question as well just to be sure. do we need this event [here](https://github.com/Qbox-project/qbx-vehicleshop/pull/32/files#diff-6f0b2f6b34829d0f65728f2508a4f5e2ef0d39d59b554b3ec8613c8feaacf74dL121)?

Comment (Manason) [reply server/main.lua:426]: No

**Review by Manason — APPROVED** [review]

**Pull request merged**